### ISSUE TITLE
Clarify how ResizeObserver interacts with fragmented elements

### ISSUE THREAD

**Issue by dholbert** [WICG/ResizeObserver#11]
Suppose an element is fragmented into several boxes (in a simple case, a `<span>` that gets wrapped into several lines. In more complex cases, a block fragmented in a multicol container).

What should happen if an author registers a `resizeObserver` callback for such an element?  (What is the rect in question that should be watched?)

Right now, the spec says we watch the [content rect](https://wicg.github.io/ResizeObserver/#content-rect) whose "width is content width" (and the text "content width" is [linked to the `width` CSS property definition](https://drafts.csswg.org/css-box-3/#width)).  But I'm not sure what "content width" actually means in the scenarios I've described.  (e.g. in multicol, is it the width of a single column? or the width of the bounding-box that spans all of the columns?)

Also note that for a `<span>` (with `display:inline`), the `width` CSS property has no effect on the actual rendered size, too -- so we may not actually want to anchor the spec in the `width` CSS property definition.

**Comment by atotic** [comment]
CSS mentions `content rect` often, but tracking down its definition has been tough. I did link to the wrong one. I've corrected the links to point to the definition I am using in the implementation.
https://www.w3.org/TR/CSS2/visudet.html#Computing_widths_and_margins

From this definition, inline elements do not have width, so ResizeObserver does not observe their width.

The multi-column spec is silent on how should content rect be computed for multi-column layouts. I just did an empirical check. Since content rect is closely related to Element.width, I did an window.getComputedStyle($0).width, and it reports the width of the first column in Chrome and FF. That's what Chrome does for ResizeObserver too.

I'll add a note to the spec.
